test/uk/gov/hmrc/frontendtemplateprovider/FooterSpec.scala

```diff
@@ -36,17 +37,16 @@ class FooterSpec extends WordSpec with Matchers  with Results with WithFakeAppli
 
   implicit val hc = HeaderCarrier()
 
+  val fakeRequest = FakeRequest("GET", "/")
+
   override lazy val fakeApplication: Application = new GuiceApplicationBuilder()
-    .configure(Map("assets.url" -> "www.example.com/"))
-    .bindings(bindModules:_*)
+    .configure(Map("assets.url" -> "www.example.com/", "assets.version" -> "1"))
     .build()
 
-  val fakeRequest = FakeRequest("GET", "/")
-
   // ToDo: Fix the hardcoded assets version in this test
   "Footer" should {
     "contain links to assets-frontend JS" in new Setup {
-      bodyText should include("<script src=\"www.example.com/2.246.0/javascripts/application.min.js\" type=\"text/javascript\"></script>")
+      bodyText should include("<script src=\"www.example.com/1/javascripts/application.min.js\" type=\"text/javascript\"></script>")
```

Comment:
Would it be possible here to build the expected value string from the values found in the configuration?